src/DSharp.Sdk/Sdk/Sdk.targets

```diff
@@ -1,53 +1,12 @@
-<Project>
-
+<Project TreatAsLocalProperty="TaskFolder">
+  
   <PropertyGroup>
+    <TaskFolder Condition=" '$(MSBuildRuntimeType)' == 'Core' ">netstandard2.0</TaskFolder>
+    <TaskFolder Condition=" '$(MSBuildRuntimeType)' != 'Core' ">net461</TaskFolder>
```

Comment:
Still says net461

Reply:
This condition is to locate the build tasks depending on whether we are running from msbuild (net framework) or dotnet build (netstandard). All DSharp assemblies are still built against 461